variants/particle_xenon/variant.h

```diff
@@ -175,13 +191,17 @@ static const uint8_t SCK1  = PIN_SPI1_SCK;
  */
 #define WIRE_INTERFACES_COUNT 2
 
-#define PIN_WIRE_SDA         _PINNUM(0, 26)
-#define PIN_WIRE_SCL         _PINNUM(0, 27)
-#define PIN_WIRE1_SDA        _PINNUM(1, 1)
-#define PIN_WIRE1_SCL        _PINNUM(1, 2)
+#define PIN_WIRE_SDA               0
+#define PIN_WIRE_SCL               1
+#define PIN_WIRE1_SDA              2
+#define PIN_WIRE1_SCL              3
 
 // On-board QSPI Flash
-#define EXTERNAL_FLASH_DEVICES   GD25Q16C
+#define EXTERNAL_FLASH_DEVICES   MX25L3233F
+
+#define USB_MSC_BLOCK_SIZE    512
+#define USB_MSC_BLOCK_COUNT   ((2*1024*1024) / USB_MSC_BLOCK_SIZE)
```

Comment:
It looks like these definitions were recently removed in 3d49611c004e0569ce2b10c7e312803bffc69183, so we may not need them here.

Reply:
@jaswope yeah right, I will remove that later. Flash device is dynamically detected now.